package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "@digitalbazaar/credentials-v2-context",
+  "version": "0.0.1-0",
+  "description": "A Verifiable Credentials v2 context library for JavaScript.",
+  "main": "js/index.js",
+  "module": "./dist/context.esm.js",
+  "files": [
+    "contexts",
+    "dist/context.js",
+    "js"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/digitalbazaar/credentials-v2-context"
+  },
+  "bugs": {
+    "url": "https://github.com/digitalbazaar/credentials-v2-context/issues"
+  },
+  "author": {
+    "name": "Digital Bazaar, Inc.",
+    "email": "support@digitalbazaar.com",
+    "url": "https://digitalbazaar.com/"
+  },
+  "homepage": "https://github.com/digitalbazaar/credentials-v2-context#readme",
+  "license": "BSD-3-Clause",
+  "devDependencies": {
```

Comment:
102MB of dependencies... for a context file. Yikes.

Reply:
Dev Deps are not in the production build provided that is where the 102 MB comment comes from. Only the base deps are in the package npm will serve.